src/components/portal/portal.tsx

```diff
@@ -44,38 +44,38 @@ export class Portal {
      * Decides which direction the portal content should open.
      */
     @Prop({ reflect: true })
-    public openDirection: OpenDirection = 'bottom';
+    public openDirection?: OpenDirection = 'bottom';
```

Comment:
Does it really matter adding `?` here? It seems unnecessary.
`openDirection= 'bottom'` when no value is passed to this component anyway. The default values here can be overridden by specifying a value for a prop with a default value.
The same comment for the changes on 53, 65, 71, 78 and 87

Reply:
Yeah, reading the typescript docs [here](https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties), it seems like `?` should be used when a property is optional but there is _not_ a default value. So I guess this is unnecessary, yes. I'll remove it again 👍

Reply:
⚡

I dropped this commit entirely 👍